sdk/entra/WebJobs.Extensions.AuthenticationEvents/src/WebJobs.Extensions.AuthenticationEvents/Schema/TokenIssuanceStart/openapi.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
I'm not sure what the purpose of including these here is.   Are they part of the local development needs?

Reply:
These are legacy when we supported json schemas, I have removed.